gitpod-io/gitpod
# [WIP] Remove server connection in supervisor frontend

## Description


## Related Issue(s)

Fixes #

## How to test

- Open workspace in preview env, it should work
- Shared workspace should have no `Gitpod: Stop Workspace` command (press F1 or cmd+P in vscode browser and search stop workspace)
- Exec `window.gitpod.service` in browser DevTool Console should have nothing
- Public API in dashboard should all failed (scoped main cookie domain)

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
